### DISCUSSION TITLE
Is there any way to access global network policies(calico)using c#?

### DISCUSSION THREAD

**Discussion by srinivasmummareddy** [kubernetes-client/csharp#1056] [Ideas]
I need to get the global network policies created in my cluster. Is there any support for this functionality?

**Comment by tg123** [comment]
client.NetworkingV1.ListNamespacedNetworkPolicy

**Reply by tg123** [reply]
Did not see your title,
you can list CRD as well

**Comment by IvanJosipovic** [comment]
You can also try my project which has the models [KubernetesCRDModelGen.Models.projectcalico.org](https://www.nuget.org/packages/KubernetesCRDModelGen.Models.projectcalico.org)

[GlobalNetworkPolicy](https://www.fuget.org/packages/KubernetesCRDModelGen.Models.projectcalico.org/1.0.0-alpha.37/lib/netstandard2.0/KubernetesCRDModelGen.Models.projectcalico.org.dll/KubernetesCRDModelGen.Models.crd.projectcalico.org/GlobalNetworkPolicy)
